packages/@react-types/button/src/index.d.ts

```diff
@@ -45,13 +45,13 @@ export interface LinkButtonProps {
 
 interface AriaBaseButtonProps extends FocusableDOMProps, AriaLabelingProps {
   /** Indicates whether the element, or another grouping element it controls, is currently expanded or collapsed. */
-  'aria-expanded'?: boolean,
+  'aria-expanded'?: boolean | 'true' | 'false',
```

Comment:
hmm... are we sure about these? why do they need to be strings?

Reply:
yeahhh, unfortunately this is needed to match the aria definitions in `@types/react` for `HTMLAttributes<HTMLElement>` -> `AriaAttributes` otherwise lint complains:

![image](https://user-images.githubusercontent.com/9661127/95246964-17433200-07ca-11eb-8803-d43fb408c0b1.png)

Reply:
Hmm that's annoying. Pretty confusing to have all these ways of doing the same thing. I fixed by using a more narrow type for `useLabels` instead, rather than all HTMLAttributes. https://github.com/adobe/react-spectrum/pull/1014/commits/29722cd8d8279379c49633e9c593808ef28bc371